### PR TITLE
Setup django-decouple

### LINKED ISSUE #873: Configure django-decouple instead of environment variables
## Description
As a Member,
I need  to configure django-decouple
so that I can eliminate the hassle of environment variables.

## Mocks
Generally django uses environment variables to set various secrets in settings.py of the project like Database creds, Secret Keys, Emailing ID details, allowed hosts, debug status.
Using environment variables is usually a hassle and is prone to error.
Django decouple takes the settings parameters from a .env file, hence even developers don't need to export environment variables to work and can simply use an example .env file.
Similarly even on the production server, this will simplify things, making it faster to deploy.
 [Read More](https://pypi.org/project/python-decouple/)
[Read More](https://medium.com/@nithinkvijayan/https-medium-com-nithinkvijayan-separating-django-application-config-and-secrets-from-code-python-decouple-e0787d2bcc2a)

### Update [Required]
- Add django-decouple to requirements.txt
- Make all fields in settings.py dynamic with django-decouple
- modify travis.yml accordingly

### Enhancement to Update [Optional]
- [ ]  Write tests

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentor.

## Estimation
1 hours

### PR DESCRIPTION
**Pull request by anshulahuja98**
# Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #873 
# Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- Outreach
- Documentation

**Code/Quality Assurance Only**
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
Tested locally but without docker image

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: vms/.env.example [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+DB_NAME=vms
+DB_USER=vmsadmin
+DB_PASSWORD=0xdeadbeef
+
+# Email configs
+
+EMAIL_HOST=localhost
+EMAIL_PORT=1025
+
+# other configs
```
Comment (Monal5031) [inline vms/.env.example]: Allowed hosts?

File: vms/vms/settings.py [hunk 2]
```diff
@@ -135,13 +126,12 @@
...
+
+EMAIL_HOST = config('EMAIL_HOST', default='localhost')
+EMAIL_PORT = config('EMAIL_PORT', default='1025')
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 LOGIN_REDIRECT_URL = reverse_lazy('home:index')
 RECOVER_ONLY_ACTIVE_USERS = False
 ACCOUNT_ACTIVATION_DAYS = 2
 ANONYMOUS_USER_ID = -1
-
```
Comment (Monal5031) [inline vms/vms/settings.py]: :thinking:

File: vms/.env.example (hunk 1 above)
Comment (anshulahuja98) [reply vms/.env.example]: Added now

File: vms/vms/settings.py (hunk 2 above)
Comment (anshulahuja98) [reply vms/vms/settings.py]: What's the issue here?

**Pull request closed without merging**